runtime/ftplugin/gitconfig.vim

```diff
@@ -13,3 +13,29 @@ setlocal formatoptions-=t formatoptions+=croql
 setlocal comments=:#,:; commentstring=;\ %s
 
 let b:undo_ftplugin = "setl fo< com< cms<"
+
+if !has('unix')
+  finish
+endif
+
+if !has('gui_running')
+  command! -buffer -nargs=1 Sman
+        \ silent exe '!' . 'LESS= MANPAGER="less --pattern=''^\s+' . KeywordLookup_gitconfig() . <q-args> . '\b'' --hilite-search" man ' . 'git-config' |
+        \ redraw!
+elseif has('terminal')
+  command! -buffer -nargs=1 Sman
+        \ silent exe 'term ' . 'env LESS= MANPAGER="less --pattern=''' . escape('^\s+' . KeywordLookup_gitconfig() . <q-args> . '\b', '\') . ''' --hilite-search" man ' . 'git-config'
+else
+  finish
```

Comment:
Using `finish` here is likely to be confusing for future modifications; editors would need to realize they must add code above this block to guarantee it always executes.